api/migrations/20180726153748_files-and-relations.js

```diff
@@ -0,0 +1,28 @@
+exports.up = async knex => {
+  await knex.schema.createTable('files', table => {
+    table.increments();
+    table.text('key');
+    table.integer('size');
```

Comment:
Should `size` be a distinct column, or should we wrap it into the metadata?  One nice thing about having it be distinct is that we can be sure it'll exist.  If we put it in metadata, there's no longer a guarantee that it exists because the metadata field is free-form.

Reply:
i like it distinct